specs/lib/writeBuffersSpec.js

```diff
@@ -39,27 +41,39 @@ describe('writeBuffers', function() {
 
     it('writes an external buffer', function(done) {
         var gltf = clone(testGltf);
+        var options = {
+            outputPath : outputPath,
+            embed : false,
+            embedImage : false,
+            createDirectory : true
+        };
 
-        writeGltf(gltf, outputPath, false, false, true, function() {
-            expect(gltf.buffers.CesiumTexturedBoxTest.extras).not.toBeDefined();
-            expect(gltf.buffers.CesiumTexturedBoxTest.uri).toEqual('CesiumTexturedBoxTest.bin');
-            fs.readFile(outputBufferPath, function(err, outputData) {
-                if (err) {
-                    throw err;
-                }
-                expect(bufferEqual(outputData, bufferData)).toBe(true);
-                done();
-            });
+        writeGltf(gltf, options)
+            .then(function() {
+                expect(gltf.buffers.CesiumTexturedBoxTest.extras).not.toBeDefined();
+                expect(gltf.buffers.CesiumTexturedBoxTest.uri).toEqual('CesiumTexturedBoxTest.bin');
+                readFile(outputBufferPath)
+                    .then(function(outputData) {
+                        expect(bufferEqual(outputData, bufferData)).toBe(true);

```

Comment:
@lasalvavida 
I appreciate the quick responses to these questions. That makes a lot of sense but I seemed to have just been gleaning over this fact while reading the documentation.